src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -1763,106 +1763,101 @@ class Organization: ADOSVTBase
 
     hidden [ControlResult] CheckMinPCACount([ControlResult] $controlResult)
     {
-        $TotalPCAMembers=0
-        $PCAMembers = @()
-        $PCAMembers += [AdministratorHelper]::GetTotalPCAMembers($this.OrganizationContext.OrganizationName)
-        $TotalPCAMembers = ($PCAMembers| Measure-Object).Count
-        $controlResult.AddMessage("There are a total of $TotalPCAMembers Project Collection Administrators in your organization.")
-        if ($this.graphPermissions.hasGraphAccess)
-        {
-            $SvcAndHumanAccounts = [IdentityHelpers]::DistinguishHumanAndServiceAccount($PCAMembers, $this.OrganizationContext.OrganizationName)
-            $HumanAcccountCount = ($SvcAndHumanAccounts.humanAccount | Measure-Object).Count
-            if($HumanAcccountCount -lt $this.ControlSettings.Organization.MinPCAMembersPermissible){
-                $controlResult.AddMessage([VerificationResult]::Failed,"Number of human administrators configured are less than the minimum required administrators count: $($this.ControlSettings.Organization.MinPCAMembersPermissible)");
-            }
-            else{
-                $controlResult.AddMessage([VerificationResult]::Passed,"Number of human administrators configured are more than the minimum required administrators count: $($this.ControlSettings.Organization.MinPCAMembersPermissible)");
-            }
-            if($TotalPCAMembers -gt 0){
-                $controlResult.AddMessage("Verify the following Project Collection Administrators: ")
-                $controlResult.AdditionalInfo += "Total number of Project Collection Administrators: " + $TotalPCAMembers;
-            }
-
-            if (($SvcAndHumanAccounts.humanAccount | Measure-Object).Count -gt 0) {
-                $humanAccounts = $SvcAndHumanAccounts.humanAccount | Select-Object displayName, mailAddress
-                $controlResult.AddMessage("`nHuman Administrators: $(($humanAccounts| Measure-Object).Count)", $humanAccounts)
-                $controlResult.SetStateData("List of human Project Collection Administrators: ",$humanAccounts)
+        $controlResult.VerificationResult = [VerificationResult]::Failed
```

Comment:
CheckMinPCACount and CheckMaxPCACount both have same implementation until the check of controlsetting is made. Can we skip the reevaluation if already done by previous control

Reply:
Skipped the SvcAndHumanAccounts api call when run by second control